core/src/main/scala/algebra/lattice/DeMorgan.scala

```diff
@@ -0,0 +1,55 @@
+package algebra
+package lattice
+
+import scala.{specialized => sp}
+
+/**
+  * De Morgan algebras are bounded lattices that are also equipped with
+  * an involutive `complement` operation.
+  *
+  * Complementation obeys the following law:
+  *
+  *  - ¬¬a = a
+  *  - ¬(x∧y) = ¬x∨¬y
+  *
+  * However, in De Morgan algebras this operation is only a
+  * pseudo-complement, since De Morgan algebras do not necessarily
```

Comment:
The definition of pseudocomplement requires `(a ∧ ¬a) = 0`, which, as you mention below, does not necessarily hold for De Morgan algebras.